docs/faq.md

```diff
@@ -7,27 +7,50 @@ Please read [Retain Client IPAddress Guide here](./user-guide/retaining-client-i
+validation will limit the characters accepted on the field "`ingress.spec.rules.http.paths.path`",
+to "`alphanumeric characters`", and  `"/," "_," "-."` Also, in this case,
+the path should start with `"/."`
 
-- When the ingress resource path contains other characters (like on rewrite configurations), the pathType value should be "`ImplementationSpecific`".
+- When the ingress resource path contains other characters (like on rewrite
+configurations), the pathType value should be "`ImplementationSpecific`".
 
 - API Spec on pathType is documented [here](https://kubernetes.io/docs/concepts/services-networking/ingress/#path-types)
 
-- When this option is enabled, the validation will happen on the Admission Webhook. So if any new ingress object contains characters other than  "`alphanumeric characters`", and  `"/," "_," "-."` , in the `path` field, but is not using `pathType` value as `ImplementationSpecific`, then the ingress object will be denied admission.
+- When this option is enabled, the validation will happen on the Admission
+Webhook. So if any new ingress object contains characters other than
+alphanumeric characters, and, `"/,","_","-"`, in the `path` field, but
+is not using `pathType` value as `ImplementationSpecific`, then the ingress
+object will be denied admission.
 
-- The cluster admin should establish validation rules using mechanisms like "`Open Policy Agent`", to validate that only authorized users can use ImplementationSpecific pathType and that only the authorized characters can be used. [The configmap value is here](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#strict-validate-path-type)
+- The cluster admin should establish validation rules using mechanisms like
+"`Open Policy Agent`", to validate that only authorized users can use
+ImplementationSpecific pathType and that only the authorized characters can be
+used. [The configmap value is here](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#strict-validate-path-type)
 
 - A complete example of an Openpolicyagent gatekeeper rule is available [here](https://kubernetes.github.io/ingress-nginx/examples/openpolicyagent/)
 
-- If you have any issues or concerns, please do one of the following: 
-  - Open a GitHub issue 
+- If you have any issues or concerns, please do one of the following:
+  - Open a GitHub issue
   - Comment in our Dev Slack Channel
-  - Open a thread in our Google Group ingress-nginx-dev@kubernetes.io
+  - Open a thread in our Google Group <ingress-nginx-dev@kuberne>
+
+## Why is chunking not working since controller v1.10 ?
+
+- If your code is setting the HTTP header `"Transfer-Encoding: chunked"` and
+the controller log messages show an error about duplicate header, it is
+because of this change <http://hg.nginx.org/nginx/rev/2bf7792c262e> . More
+details are available in this issue https://github.com/kubernetes/ingress-nginx/issues/11162
```

Comment:
```suggestion
If your code is setting the HTTP header `"Transfer-Encoding: chunked"` and the controller log messages show an error about duplicate header, it is because of this change: <http://hg.nginx.org/nginx/rev/2bf7792c262e>.

More details are available in this issue <https://github.com/kubernetes/ingress-nginx/issues/11162>.
```

Reply:
fixed as suggested

Reply:
I suggested not to have them as list items. But that might be petty. In the end and if we use list items, the both lines should be on the same line.

Reply:
I am ok with a list of 2 bullets under 1 level2 heading. if that is fine with you.

Reply:
Yes, it is! I just didn't want to have a single bullet paragraph.